src/features/courses/course-route.ts

```diff
@@ -603,6 +603,57 @@ router.post('/question',
         }
     }));
 
+router.get('/question/:id/raw',
+    authenticationMiddleware,
+    validate(getQuestionRawValidation),
+    // This is a typescript workaround since it tries to use the type extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    asyncHandler(async (req: RederlyExpressRequest<any, unknown, GetQuestionRawRequest.body, unknown>, _res: Response, next: NextFunction) => {
+        if (_.isNil(req.session)) {
+            throw new Error(Constants.ErrorMessage.NIL_SESSION_MESSAGE);
+        }
+
+        const { id: questionId } = req.params as GetQuestionRawRequest.params;
+        const { userId } = req.query as GetQuestionRawRequest.query;
+        
+        try {
```

Comment:
Async handler was added after the fact so we have been moving away from doing try catches since it automatically calls next